go.mod

```diff
@@ -70,4 +71,7 @@ require (
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 )
 
-replace golang.org/x/crypto => github.com/cli/crypto v0.0.0-20210929142629-6be313f59b03
+replace (
+	github.com/imdario/mergo => github.com/heaths/mergo v0.3.13-0.20220516084937-200f90d97053
```

Comment:
@mislav this is my branch for imdario/mergo#210. Perhaps you want to fork to cli/mergo or something like you did for the crypto library below. It doesn't look like mergo has seen any active development for a while, but after reviewing a number of JSON merge utilities, this seemed like the best one that wasn't focused on json+patch.